staging/src/k8s.io/apimachinery/pkg/util/proxy/transport_test.go

```diff
@@ -274,3 +277,84 @@ func TestProxyTransport(t *testing.T) {
 		testItem(name, &item)
 	}
 }
+
+func TestRewriteResponse(t *testing.T) {
+	gzipbuf := bytes.NewBuffer(nil)
+	flatebuf := bytes.NewBuffer(nil)
+
+	testTransport := &Transport{
+		Scheme:      "http",
+		Host:        "foo.com",
+		PathPrepend: "/proxy/node/node1:10250",
+	}
+	expected := []string{
+		"short body test: gzip test",
```

Comment:
last set of comments, sorry :)

1) remove gzip reference :)
2) suggest making a realllllly big one with e.g. `strings.Repeat("a",4097)`
3) please squash, thanks.

Reply:
@lavalamp Thanks, squashed and pushed.

Reply:
thanks, just waiting for tests now.

Reply:
@lavalamp The tests all passed now.